kernel/data-pipeline/core/src/main/java/org/apache/shardingsphere/data/pipeline/common/sqlbuilder/PipelineImportSQLBuilder.java

```diff
@@ -61,11 +71,11 @@ public PipelineImportSQLBuilder(final DatabaseType databaseType) {
      */
     public String buildInsertSQL(final String schemaName, final DataRecord dataRecord) {
         String sqlCacheKey = INSERT_SQL_CACHE_KEY_PREFIX + dataRecord.getTableName();
-        if (!sqlCacheMap.containsKey(sqlCacheKey)) {
+        if (null!=sqlCache.getIfPresent(sqlCacheKey)) {
```

Comment:
There's some code style could be improved. Refer to [Code of Conducts]( https://shardingsphere.apache.org/community/en/involved/conduct/code/ ) for more details, and format code.

Reply:
Thanks. I try to refer